### PR TITLE
remove levelCounts in Factor

### PR DESCRIPTION
**Pull request by Huandao0812**
see https://github.com/ddf-project/DDF/pull/281

### CODE REVIEW

**Comment by hai-adatao** [comment]
@nhanitvn @lebinh please help review

**Comment by nhanitvn** [comment]
lgtm

File: aws/src/main/scala/io/ddf/aws/ml/MLMetricsSupporter.scala [hunk 1]
```diff
@@ -24,12 +24,8 @@ class MLMetricsSupporter(ddf: DDF) extends io.ddf.ml.AMLMetricsSupporter(ddf) {
   }
 
   @throws(classOf[DDFException])
-  override def roc(predictionDDF: DDF, alpha_length: Int): RocMetric = {
-    val originalDDF = ddf
-    val model = ddf.ML.train("BINARY")
-    //wait for model
-    val predictedDDF: DDF = predictionDDF.ML.applyModel(model)
-    val predictDDFAsSql = predictedDDF.getRepresentationHandler.get(Representations.SQL_ARRAY_RESULT)
+  override def roc(alpha_length: Int): RocMetric = {
```
Comment (lebinh) [inline aws/src/main/scala/io/ddf/aws/ml/MLMetricsSupporter.scala:27]: Why is this param in snake_case?

**Comment by lebinh** [comment]
@Huandao0812 look like you are removing a lot of tests, why?

**Comment by Huandao0812** [comment]
I will put back some of the test cases

**Pull request merged**